scrapy/core/downloader/contextfactory.py

```diff
@@ -135,11 +135,11 @@ def load_context_factory_from_settings(settings, crawler):
             settings=settings,
             crawler=crawler,
         )
-        msg = """
-            '%s' does not accept `method` argument (type OpenSSL.SSL method,\
-            e.g. OpenSSL.SSL.SSLv23_METHOD) and/or `tls_verbose_logging` argument and/or `tls_ciphers` argument.\
-            Please upgrade your context factory class to handle them or ignore them.""" % (
-            settings['DOWNLOADER_CLIENTCONTEXTFACTORY'],)
+        msg = f"""
+        {str(settings['DOWNLOADER_CLIENTCONTEXTFACTORY'],)} \
```

Comment:
I think we don't need a tuple here, not sure why it was used before though.

Reply:
@wRAR,  fixed -> https://github.com/scrapy/scrapy/pull/5246/commits/2a1faec1dabee5011698c5aade263dfa378c11f6